source/extensions/common/opentelemetry/sdk/common/types.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <map>
+#include <string>
+
+#include "opentelemetry/common/attribute_value.h"
+#include "opentelemetry/proto/common/v1/common.pb.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Common {
+namespace OpenTelemetry {
+namespace Sdk {
+namespace Common {
```

Comment:
I'd focus on these instead - these seem to be reused a lot. I'd recommend flattening the namespace, to just `OpenTelemetry` or `Otel`.